server/model/user.py

```diff
@@ -71,4 +77,14 @@ def delete_user(username):
     db.session.delete(user)
     db.session.commit()
 
-    return user
\ No newline at end of file
+    return user
+
+def add_completed_assignment(data):
```

Comment:
I see that it's adding to to an assignment list, but can't see where that assignment list exists in the User model.

Reply:
Also delete this